plaid/plaid.go

```diff
@@ -30,39 +29,38 @@ type Client struct {
 	httpClient  *http.Client
 }
 
+type ClientOptions struct {
```

Comment:
can we add some comment here stating that `HttpClient` is optional? 

Also, it may need to be `HTTPClient` to satisfy the linter? Usually acronyms in golang are uppercased.

Reply:
disregard, looks like this has been resolved already ⚡️